### DISCUSSION TITLE
3.4.5 release

### DISCUSSION THREAD

**Discussion by kjk** [sumatrapdfreader/sumatrapdf#2672] [Announcements]
A minor release:

* fixes crashes

<hr /><em>This discussion was created from the release <a href='https://github.com/sumatrapdfreader/sumatrapdf/releases/tag/3.4.5rel'>3.4.5 release</a>.</em>

**Comment by antdude** [comment]
Uh, what happened to https://www.sumatrapdfreader.org/news?

**Reply by kjk** [reply]
Fixed, thanks.

**Comment by Ronald-G-Andrade** [comment]
Thank you very much, I really like this App 👍🏻  It's so lightweight and fast, my best whishes to you, @kjk  😃 !!

**Comment by antdude** [comment]
You're welcome, kjk. What happened?

**Reply by GitHubRulesOK** [reply]
I suspect it is down to a Cloudflare anti mail scrapper redirect, but that is conjecture.

**Comment by RayReddingtonx** [comment]
It's still taking 3 seconds to load up all my opened files. I will simply go back to the older version.

**Reply by kjk** [reply]
Minor releases only fix crashes and serious bugs so don't expect any new features or speed improvements.
